test/framework/src/main/java/org/elasticsearch/test/rest/support/Features.java

```diff
@@ -34,7 +34,7 @@
  */
 public final class Features {
 
-    private static final List<String> SUPPORTED = Arrays.asList("stash_in_path", "groovy_scripting", "headers", "embedded_stash_key");
+    private static final List<String> SUPPORTED = Arrays.asList("stash_in_path", "groovy_scripting", "headers", "embedded_stash_key", "yaml");
```

Comment:
This has a line-length violation. I pushed c49c67ec589c949fd4802eb65e2a7b01b411ce91.